cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -213,18 +213,21 @@ func textFormatPrintMetric(obj *api.Metric) {
 }
 
 func textFormatPrintRepoGC(obj *api.RepoGC) {
-	if obj.Error != "" {
-		fmt.Printf("  > Error: %s\n", obj.Error)
-	}
-
 	fmt.Printf("  > CIDs:\n")
 	for _, key := range obj.Keys {
-		fmt.Printf("    - %s\n", key.String())
+		if key.Error != "" {
+			fmt.Printf("    - %s | ERROR: %s\n", key.Key.String(), key.Error)
```

Comment:
On going through related IPFS code to see how this happens.

One more(important) thing that I have missed is that, `/repo/gc` api would keep error empty if not passed `stream-errors`. This can be seen in code
https://github.com/ipfs/go-ipfs/blob/cd2611db377fecca28140fa1723c7a5a13295e0a/core/commands/repo.go#L79-L106 

I tried to confirm this by manually adding an error, running with `?stream-errors=true`, but it would just get stuck instead of provide output.

I would pass this argument while making calls to IPFS so that we can see errors.